LManhL/TKXDPM.KHMT.20231-15
# Feature: New UI and base class for admin screen

## Description
- This PR creates new UI for some functions for Admin page.
- Also create (properly) all the (base) class for admin page.

## Impact range:
- None

## Logic
- The UI(s) were created by SceneBuilder.

## Picture
![image](https://github.com/LManhL/TKXDPM.KHMT.20231-15/assets/119603933/191e7a38-e3c8-4d66-b889-d6167093526f)
![image](https://github.com/LManhL/TKXDPM.KHMT.20231-15/assets/119603933/9ee65a01-ffa6-43ea-b4c1-5eba89cf1392)
![image](https://github.com/LManhL/TKXDPM.KHMT.20231-15/assets/119603933/09024881-1a72-406f-adb2-58441d3cf5b5)


## How to test this?
- Go into the folder views => right click on the file => Open with SceneBuilder.
- And you can also check for the new code in the entity folder.
